Gopkg.toml

```diff
@@ -14,12 +14,14 @@ required = [
     "sigs.k8s.io/controller-runtime/pkg/source",
     "sigs.k8s.io/testing_frameworks/integration", # for integration testing
     "sigs.k8s.io/kustomize",
+    "github.com/golang/dep/cmd/dep",
+    "golang.org/x/tools/cmd/goimports", # for testing
```

Comment:
goreturns is the default in VSCode, let's consider using that (but we don't have to).

Reply:
Lets tackle that at a later point.